src/test/java/com/training/services/tests/GuestAccountAuthenticateTest.java

```diff
@@ -0,0 +1,37 @@
+package com.training.services.tests;
+
+import com.training.base.BaseTest;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.json.simple.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import services.ga.AuthenticationResponse;
+import static io.restassured.RestAssured.given;
+
+public class GuestAccountAuthenticateTest extends BaseTest{
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void  testGuestAuthenticate(){
+        RestAssured.baseURI = map.get("URI");
+        AuthenticationResponse authenticationResponse = given().
+                header(map.get("AppKeyHeader"), (map.get("AppKeyValue"))).
+                header(map.get("ContentTypeHeader"), (map.get("ContentTypeValue")))
+                .body("{\n" +
+                        " \"uid\": \"testShrikant@api.com\",\n" +
+                        " \"password\": \"Password1\"\n" +
+                        "}").post("/authentication/login")
+                .then().extract().response()
+                .as(AuthenticationResponse.class);
```

Comment:
Can you make this more readable by doing each functions in separate lines, like :
then()
.extract()
.response()
.as(Authentication.class)